src/data/team.js

```diff
@@ -0,0 +1,133 @@
+export const teamData = [
+{
+    id: 1,
+    firstName: 'Alex',
+    lastName: 'Gutierrez',
+    position: 'React Apprenticeship Lead',
+    email: 'agutierrez@bitwiseindustries.com',
+    linkedIn: 'https://www.linkedin.com/in/alex-gutierrez-mpa-6ba8bb34/',
+    github: 'https://github.com/alexg1021',
+    bio: 'lorem ipsum',
+    profile_img: 'https://media-exp1.licdn.com/dms/image/C5603AQH7DxGvg3R3XA/profile-displayphoto-shrink_200_200/0/1536703041727?e=1622678400&v=beta&t=eKhvifwFxw7KfJn2_a2OzUXVuixmsa8JlgPqo6YjuBY',
+    languages: ['html','css', 'javascript']
+},
+{
+    id: 2,
+    firstName: 'Anna',
+    lastName: 'Jimenez',
+    position: 'React Apprentice',
+    email: 'annamariej1015@gmail.com',
+    linkedIn: 'https://www.linkedin.com/in/annamarie-jimenez-231b79182/',
+    github: 'https://github.com/annamariej1015',
+    bio: 'lorem ipsum',
+    profile_img: 'https://media-exp1.licdn.com/dms/image/C5603AQEH3UH2jdXzdQ/profile-displayphoto-shrink_400_400/0/1611110839376?e=1622678400&v=beta&t=zOKhMYmgPrDbPa4bZOIrHyp2S4Y-1uRTIgFwNmA1tgs',
+    languages: ['html', 'css', 'javascript', 'react', 'react native', 'nodeJS', 'mongoDB', 'express']
+}, 
+{
+    id: 3,
+    firstName: 'Brittney',
+    lastName:'Cavazos',
+    position: 'React Apprentice',
+    email:'bcavazos@alphaworks.tech',
+    linkedIn:'https://www.linkedin.com/in/brittney-cavazos-83b056a9/',
+    github:'https://github.com/b-cavazos',
+    bio:'lorem ipsum',
+    profile_img:'https://media-exp1.licdn.com/dms/image/C5603AQHRFP4gIJp58g/profile-displayphoto-shrink_800_800/0/1578492310711?e=1622678400&v=beta&t=0HRaphHlpZn0giNFr799ZJyqimnglvrWnmKaKM-E9aM',
+    languages:['html','css','javascript', 'react']
+}, 
+{
+    id: 4,
+    firstName: 'Elizabeth',
+    lastName: 'Gaw',
+    position: 'React Apprentice',
+    email: 'hello.xyeras@gmail.com',
+    linkedIn: 'https://www.linkedin.com/in/elizabeth-agnes-gaw-6702a7139/',
+    github: 'https://github.com/xyeras',
+    bio: 'lorem ipsum',
+    profile_img: 'https://media-exp1.licdn.com/dms/image/C5603AQF0ROaUm_EXWw/profile-displayphoto-shrink_400_400/0/1611397157383?e=1622678400&v=beta&t=ajryz6YmZznohK-PD6Ltq63ExXHWUwm7wY54oqyO2BA',
+    languages: ['html','css', 'javascript', 'react', 'python', 'haskell', 'oz', 'c++']
+},
+{
+    id: 5, 
+    firstName: 'Harrsion',
+    lastName: 'Stamps',
+    email: 'hstamps4@gmail.com',
```

Comment:
Harrison's position property is missing

Reply:
His data came in right as I was uploading it & I copy-pasted it.  Good catch.